### PR TITLE
Clean up testing mock data and class name usage.

### LINKED ISSUE #238: Remove usage of `.shepherd-transparent-text` class
The `shepherd-transparent-text` CSS class is used pretty liberally throughout the test dummy app, and it's even used as a default class for the "error" step we add in the `tour` service. 

However, I don't think this class is actually a part of the styles for `shepherd`, and we don't have any style rules for it here. It seems to be limited to [`shepherd`'s test dummy app styling](https://github.com/shipshapecode/shepherd/search?q=shepherd-transparent-text&unscoped_q=shepherd-transparent-text). 

If that's the case, I think we should get rid of it here to eliminate some confusion.

### PR DESCRIPTION
**Pull request by BrianSipple**
Closes #238

### CODE REVIEW

File: tests/acceptance/ember-shepherd-test.js [hunk 1]
```diff
@@ -250,49 +185,39 @@ module('Acceptance | Tour functionality tests', function(hooks) {
...
-        copyStyles: false,
-        highlightClass: 'highlight',
-        title: 'Welcome to Ember-Shepherd!',
-        text: ['Testing defaults']
+    const stepsWithoutClasses = [
+      {
+        id: 'test-highlight',
+        options: {
+          attachTo: '.first-element bottom',
+          builtInButtons: [
```
Comment (rwwagner90) [inline tests/acceptance/ember-shepherd-test.js]: Shouldn't we use `builtInButtons.cancel` etc here?

File: tests/dummy/app/routes/application.js [hunk 2]
```diff
@@ -3,13 +3,13 @@ import Route from '@ember/routing/route';
...
 import { scheduleOnce } from '@ember/runloop';
-import steps from '../data';
+import { steps as defaultSteps } from '../data';
 
 export default Route.extend({
   tour: service(),
   disableScroll: true,
   defaults: {
-    classes: 'shepherd-element shepherd-theme-arrows',
+    classes: 'shepherd-element shepherd-theme-arrows custom-default-class',
```
Comment (rwwagner90) [inline tests/dummy/app/routes/application.js]: `shepherd-element` is applied by Shepherd itself, so we don't need that class anywhere.

File: tests/acceptance/ember-shepherd-test.js (hunk 1 above)
Comment (BrianSipple) [reply tests/acceptance/ember-shepherd-test.js]: Yep -- missed a few spots.

File: tests/dummy/app/routes/application.js (hunk 2 above)
Comment (BrianSipple) [reply tests/dummy/app/routes/application.js]: ✂️

File: tests/acceptance/ember-shepherd-test.js (hunk 1 above)
Comment (BrianSipple) [reply tests/acceptance/ember-shepherd-test.js]: Updated

**Pull request merged**